lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -284,6 +284,14 @@ def create(work_package, context = {})
 
         property :activities, embedded: true, exec_context: :decorator
 
+        property :revisions,
+                 embedded: true,
+                 exec_context: :decorator,
+                 if: -> (*) {
+                   current_user_allowed_to(:view_changesets,
+                                           context: represented.project)
+                 }
```

Comment:
It's a trap!

You are now only defining the (optional) `_embedded/revisions` and leave out the (mandatory) `_links/revisions`.